src/rocker/core.py

```diff
@@ -114,7 +115,11 @@ def build(self, **kwargs):
                 else:
                     print("no more output and success not detected")
                     return 2
- 
+            except requests.exceptions.ConnectionError as ex:
+                print("Docker build failed\n", ex)
+                print(ex)
+                print('Do you have permissions to run docker such as being in the docker group?')
```

Comment:
I can't get this check or the one in `pull` to trigger from the CLI. I think the check in `main()` is sufficient.